setup_testing_node.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash -e
+
+if ! [ $(id -u) = 0 ] ; then
+	echo "$0 must be run as sudo user or root"
+	exit 1
+fi
+
+#We need to work around these default ip ranges, to make sure the tests pass
+#192.168.7.x = default usb0
+#192.168.8.x = default SoftAP0
+#192.168.9.x = (bbxm, work around 192.168.7.x usb0)
```

Comment:
Why specifically call out BeagleBoard-xM here? I think the main issue is just that the BeagleBone's were already getting modified for this issue to use as a USB host. Would be nice if we could come up with a more generic scheme such that collisions could be avoided and boards could be plugged into each other by default.

Anyway, my specific comment here is why isn't this just an alternate usb0 setting per the comment and why is 192.168.10.x also added?